tool-plugins/intellij/getting-started/plugin-features/README.md

```diff
@@ -12,53 +30,23 @@ You can debug Ballerina main/service programs with only few clicks as well.
 
 ![alt text](images/DebugSupport.gif)
 
-## Import packages on the fly
+## Import modules on the fly
 
-This feature adds import declarations on the fly. All you need to do is selecting the package name from the lookup list and the package declaration will be added automatically.
+This feature adds import declarations on the fly. All you need to do is selecting the module name from the lookup list and the module declaration will be added automatically.
 
 ![alt text](images/AutoImports.gif)
 
-## Auto import unambiguous packages
+## Auto import unambiguous modules
 
-When we copy and paste some Ballerina codes, this feature will automatically import unambiguous imports. Then we can manually import any packages with ambiguity using clicking on the package name and pressing `Alt + Enter` keys.
+When we copy and paste some Ballerina codes, this feature will automatically import unambiguous imports. Then we can manually import any modules with ambiguity using clicking on the module name and pressing `Alt + Enter` keys.
```

Comment:
Change to "When you copy and paste some Ballerina code, this feature automatically imports unambiguous imports. Then you can manually import any modules with ambiguity by clicking on the module name and pressing `Alt + Enter` keys."

Reply:
Changed